tests/ui_tests/test_bundle_list_page.py

```diff
@@ -41,6 +52,51 @@ def page(app_fs: ADCMTest, login_to_adcm_over_api) -> BundleListPage:
     return BundleListPage(app_fs.driver, app_fs.adcm.url).open()
 
 
+@pytest.fixture(params=["cluster_community"])
+@allure.title("Upload bundle")
+def upload_bundle(request: SubRequest, sdk_client_fs: ADCMClient) -> Bundle:
```

Comment:
I think you can use this fixture also for multiple upload bundles

Reply:
Multiple bundles upload is required only in pagination test and uses specially created "dummy" bundles instead of looking into the test's data directory.
It can be more general, but I don't think it'll be much of use for now

Reply:
Just I think we can use only 2 fixtures - `create_bundles` and `upload_bundles`. Parametrize `create_bundles` by `count` and `path` to bundle, create bundles inside based on a bundle from `path`

Reply:
The current implementation of `create_bundles` just spam very similar bundles, not considering any "parent" bundle.
It can be reworked to consider such a bundle or to take config dict for each bundle.
But it looks more complex.
Now we have one method to upload bundle from data dir (similar to methods in other list page tests)
and another one to test pagination avoiding restraint on unique bundle name.
Yes, more code, more fixtures, but too different use cases.

Reply:
Now this makes it difficult to understand. Fixtures do the same things, but they are different. For example, why do we use `bundle_archive`, and can't use `create_bundle_archives`?

Reply:
Both can be used, depends on what you want: take "static" data from cluster definition in datadir or "dynamic" by creating them with the same config during setup.
I think to avoid this confusion I'll better keep `create_bundle_archives` here and replace other bundle creating stuff with it. Don't think it can be avoided with renaming now.
If it can be parametrized with dict config, then it can stay as the only bundle creating method in these tests.